src/utils/common/common.tsx

```diff
@@ -195,14 +195,20 @@ export function formatExpiryDate(endTimestamp?: number) {
   );
 }
 
-export const formatIO = (io: number) => {
+export const formatARIO = (ario: number) => {
   return Intl.NumberFormat('en-US', {
     notation: 'compact',
     maximumFractionDigits: 2,
     compactDisplay: 'short',
-  }).format(io);
+  }).format(ario);
 };
 
+export function formatARIOWithCommas(ario: number) {
+  return ario.toLocaleString('en-US', {
```

Comment:
Seems odd/inconsistent we have two functions for formatting ARIO values, one with 3 digits and one with 2.

Reply:
Well its for different circumstances, one is long form one is short form. In this context we have a dropdown that we want to represent the numbers as compact as possible, and we have a larger area we want to display the numbers long form.